### DISCUSSION TITLE
Does iLEAPP have GitHub Repo, RAT, and Sideloading history?

### DISCUSSION THREAD

**Discussion by citycritter** [abrignoni/iLEAPP#610] [Q&A] [answered]
If someone deployed repos, phishing links, RATs, or sideloaded apps from sites like GitHub, would iLEAPP have the history? If so where?

**Comment by abrignoni** [comment]
I don't know what you are talking about.

**Reply by citycritter** [reply]
If someone connected or remote accessed my device using repos, phishing links, RATs, or sideloaded apps, what files should I look through using iLEAPP?

**Reply by abrignoni** [reply, marked as answer]
You are asking for an iOS malware behavioral identification course. I haven't written one yet. 

As of today iLEAPP just finds and parses artifacts from an iOS extraction. These artifacts, if properly interpreted, could be used as possible indicators of malware infection. Sadly I don't have the bandwidth to delve into that topic at this moment. Hopefully some other folks can.

Check out the Digital Forensics Discord server. There are tons of knowledgeable people there you can ask about these topics and more. 

https://github.com/Digital-Forensics-Discord-Server

**Reply by citycritter** [reply]
Ok thank you for the help, great code btw
